internal/pkg/log/log.go

```diff
@@ -1,157 +1,50 @@
 package log
 
 import (
-	"fmt"
 	"io"
-	"strings"
 
-	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 )
 
-type WriteCloser struct {
```

Comment:
`WriteCloser` -> `LevelWriter`, vid dalej.
Tato struktura implementuje interface `io.Writer` a umoznuje zapisovat lubovolny text do `logger`.
Pri vytvarani struktury sa nastavi pod akym `Level` sa ma logovat.
Pouzivame to, napr, pri logovani `pull` planu.